schema.sql

```diff
@@ -80,4 +80,10 @@ CREATE TABLE visits (
     vet_id INT REFERENCES vets(id),
     visit_date DATE,
     PRIMARY KEY (animal_id, vet_id, visit_date)
-);
\ No newline at end of file
+);
+
+--Add an email column to your owners table
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX audit_visits ON visits (animal_id, vet_id);
```

Comment:
- [ ] Kindly add an index for email, then we retry the last query to see if we can get under 1000ms